common/nns.go

```diff
@@ -0,0 +1,43 @@
+package common
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/interop"
+	"github.com/nspcc-dev/neo-go/pkg/interop/contract"
+	"github.com/nspcc-dev/neo-go/pkg/interop/convert"
+	"github.com/nspcc-dev/neo-go/pkg/interop/lib/address"
+	"github.com/nspcc-dev/neo-go/pkg/interop/native/management"
+	"github.com/nspcc-dev/neo-go/pkg/interop/native/std"
+)
+
+// NNSID is the ID of the NNS contract in NeoFS networks. It's always deployed
+// first.
+const NNSID = 1
+
+// ContractTLD is the default domain used by NeoFS contracts.
+const ContractTLD = "neofs"
+
+// InferNNSHash returns NNS contract hash by its well-known ID or panics if
```

Comment:
```suggestion
// InferNNSHash returns NNS contract hash by [NNSID] or panics if
```
it's not well-known enough